packages/smarthr-ui/src/components/NotificationBar/stories/NotificationBar.stories.tsx

```diff
@@ -0,0 +1,153 @@
+import { action } from '@storybook/addon-actions'
+import React from 'react'
+
+import { Button } from '../../Button'
+import { Stack } from '../../Layout'
+import { Text } from '../../Text'
+import { TextLink } from '../../TextLink'
+import { NotificationBar } from '../NotificationBar'
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+export const _messages = {
+  String: 'NotificationBar が表示されました',
+  ReactNode: (
+    <>
+      <Text>任意の ReactNode を設定できます</Text>
+      <Button size="s">編集</Button>
+      <TextLink href="#top">
+        <Text size="S">ヘルプ</Text>
+      </TextLink>
+    </>
+  ),
+}
+export const _onCloseHandlers = {
+  あり: () => {
+    action('onClose')()
+  },
+  なし: undefined,
+}
+
+export default {
+  title: 'STATES（状態）/NotificationBar',
+  component: NotificationBar,
+  render: (args) => (
+    <div className="shr-bg-base-grey shr-min-h-[50vh] shr-p-1">
+      <NotificationBar {...args} />
+    </div>
+  ),
+  args: {
+    base: 'none',
+    animate: true,
```

Comment:
shoud: Story のデフォルト値はコンポーネントのデフォルト値に寄せたいと思いました。

```suggestion
    animate: false,
```